cross-origin-http.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
Did you mean to commit this file?

Reply:
yes. Sorry i didn't explain: Its a bit of a pain to test SAB without a file like this, so if you want to start a cross origin isolated version of this repo you can run python3 cross-origin-http.py and it will do it for you. I can add a readme? Forgot to do that.

Reply:
(was the original reason i forked and started mucking about)

Reply:
A sentence or two in the readme would be great, yeah.

Reply:
If I remember I'll try to add my [standard fix](https://github.com/gzuidhof/coi-serviceworker) for this, which is a service worker to inject the necessary headers (so that it can still be done on Github Pages instead of needing the local server). But no promises on timeline.

Reply:
ahhh thats a nice way to do it. Well -- i can add the readme change for now and when you've got time for something more elegant you can just remove this file and the line?

Reply:
Yup, thanks!